recipe/meta.yaml

```diff
@@ -10,32 +10,54 @@ source:
   sha256: 887a6d471c901b1a6e6574ebaeeebb45e5269a79d095fe9a8f88d6614ed2e5f0
 
 build:
-  number: 0
-  noarch: python
-  entry_points:
-    - python-build = build.__main__:entrypoint
-  script: {{ PYTHON }} -m pip install . -vv
-
-requirements:
-  host:
-    - python >=3.6
-    - pip
-  run:
-    - python >=3.6
-    - colorama
-    - importlib-metadata >=0.22
-    - packaging >=19.0
-    - pep517 >=0.9.1
-    - tomli >=1.0.0
-
-test:
-  imports:
-    - build
-  commands:
-    - pip check
-    - python-build --help
-  requires:
-    - pip
+  number: 1
+
+outputs:
+  - name: {{ name }}
+    build:
+      noarch: python
+      entry_points:
+        - python-build = build.__main__:entrypoint
+        script: {{ PYTHON }} -m pip install . -vv
+
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+      run:
+        - python >=3.6
+        - colorama
+        - importlib-metadata >=0.22
+        - packaging >=19.0
+        - pep517 >=0.9.1
+        - tomli >=1.0.0
+
+    test:
+      imports:
+        - build
+      commands:
+        - pip check
+        - python-build --help
+      requires:
+        - pip
+
+
+  - name: build
```

Comment:
The reason to name it python-build was b/c the build name was too generic and we did not wanted to take that namespace. Pinging @isuruf here who suggested this in the first place. I have to confess that I don't have a strong opinion but I'd rather renamed it to `build` than maintain the alias.